tests/python/unittest/test_ndarray.py

```diff
@@ -15,15 +15,11 @@
 # specific language governing permissions and limitations
 # under the License.
 
-import os
-import mxnet as mx
-import numpy as np
 import pickle as pkl
 import unittest
 from nose.tools import raises
 from mxnet.test_utils import *
```

Comment:
Minor: Can you also expand "from mxnet.test_utils import *" so as not to use *

Reply:
resolved